User profile

malawi-mike opened the pull request.

jdepree commented:
Use this one: https://github.com/Village-X-Org/Site-v2/pull/119

jdepree commented:
Do a git fetch origin, then git checkout user-profile-b

jdepree commented:
A cherry-pick adds a given commit to a new branch.  So I just added your
commit to the branch user-profile-b

malawi-mike commented:
cool. i just added two new projects -- Manyumba teacher house and kapenda
clean water.

how's everything going?

On Mon, May 7, 2018 at 10:13 PM, Jeff DePree <notifications@github.com>
wrote:

> A cherry-pick adds a given commit to a new branch. So I just added your
> commit to the branch user-profile-b
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Village-X-Org/Site-v2/pull/118#issuecomment-387262684>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcyICu37UoIvYc9RWCuCpRf1aYGCEb4Tks5twP8-gaJpZM4Tzyxe>
> .
>



-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagex.org>
mike@villagex.org
202-360-9931
<http://www.villagex.org>
